dask/tests/test_cli.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+click = pytest.importorskip("click")
+
+from click.testing import CliRunner
+
+import dask
+from dask.cli import cli
+
+
+def test_version():
```

Comment:
It'd be good to also add test for other commands being added (e.g. `dask version`) and `register_third_party`

Reply:
I refactored the registering code a bit to make registering commands somewhat test-able